SETUP.md

```diff
@@ -12,6 +12,7 @@ You can do Code.org development using OSX, Ubuntu, or Windows (running Ubuntu in
 1. `rbenv rehash`
 1. `cd code-dot-org`
 1. `bundle install`
+1. `cp locals.yml.default locals.yml`
```

Comment:
Do you mean `rake install`?  I don't think we control what bundler does, beyond what's in our Gemfile.

The reason I haven't added this step to our install script is that it's just for a dev environment, and not something we want our servers to do.  I suppose a rake task can check the environment though...